aiohttp/helpers.py

```diff
@@ -360,6 +362,13 @@ def content_disposition_header(disptype: str,
     return value
 
 
+def is_expected_content_type(response_content_type: str,
+                             expected_content_type: str) -> bool:
+    if expected_content_type == 'application/json':
```

Comment:
I think this should go to a constant. Now it's hardcoded all over the place which is not maintainable.

Reply:
I disagree.
The string is better readable than constant.
Also, we never change this line in the future.
What maintenance problem are you talking about?

Reply:
I think, the general Python code style welcomes usage of some well-known constants as literals and not const variables. For example, everyone knows that 404 is the HTTP code for Not Found, as well as `application/json` is the name for a mime type that will not be changed in future. So I would leave it as a string literal (same as the default parameter of `content_type` in `ClientResponse.json()`)